app/controllers/Contributions.java

```diff
@@ -2244,11 +2245,21 @@ public static Contribution createContribution(Contribution newContrib,
 
         Logger.info("Using Etherpad server at: " + etherpadServerUrl);
         Logger.debug("Using Etherpad API Key: " + etherpadApiKey);
-
-        if (type != null && (type.equals(ContributionTypes.PROPOSAL) || type.equals(ContributionTypes.NOTE))) {
-            ContributionsDelegate.createAssociatedPad(etherpadServerUrl, etherpadApiKey, newContrib, t, containerResourceSpace.getResourceSpaceUuid());
-        }
-
+        
+        Campaign ca = Campaign.readByResourceSpaceId(containerResourceSpace.getResourceSpaceId());        
+        if (ca != null){
+        	List<Config> campaignConfigs = Config.findByCampaignAndKey(ca.getUuid(), "appcivist.campaign.disable-etherpad");        	
```

Comment:
- [ ] Instead than using the key directly here, let's use the class `GlobalDataConfigKeys.APPCIVIST_CAMPAIGN_DISABLE_ETHERPAD` (then if we change the key in the future, we just need to update that variable.